yarn.lock

```diff
@@ -1905,319 +1885,323 @@ __metadata:
-    gray-matter: ^4.0.2
-    loader-utils: ^2.0.0
+    gray-matter: ^4.0.3
     mdast-util-to-string: ^2.0.0
     remark-emoji: ^2.1.0
     stringify-object: ^3.3.0
     unist-util-visit: ^2.0.2
     url-loader: ^4.1.1
-    webpack: ^4.44.1
+    webpack: ^5.37.0
   peerDependencies:
     react: ^16.8.4 || ^17.0.0
     react-dom: ^16.8.4 || ^17.0.0
-  checksum: bbb16da8ca57d5b1574d7ef4fb1829eb81e823017e9640d1592a171cbcffd8581291932447f3f99d79f38d715998b2d6c026a172eac57c9407e90f7799d68689
+  checksum: 2fd7a87b19dd699397be8a6e753cfbf32ce62a72baa33fbf6847f5e723e8c98c1df87b5622d210520f1e0f6a85a0a3eb5508602a654b19f15d7f2d8dbfe91309
   languageName: node
   linkType: hard
 
-"@docusaurus/plugin-client-redirects@npm:2.0.0-alpha.c32d8bd11":
-  version: 2.0.0-alpha.c32d8bd11
-  resolution: "@docusaurus/plugin-client-redirects@npm:2.0.0-alpha.c32d8bd11"
+"@docusaurus/plugin-client-redirects@npm:^2.0.0-beta.0":
+  version: 2.0.0-beta.ff31de0ff
+  resolution: "@docusaurus/plugin-client-redirects@npm:2.0.0-beta.ff31de0ff"
   dependencies:
-    "@docusaurus/core": 2.0.0-alpha.c32d8bd11
-    "@docusaurus/types": 2.0.0-alpha.c32d8bd11
-    "@docusaurus/utils": 2.0.0-alpha.c32d8bd11
-    "@docusaurus/utils-validation": 2.0.0-alpha.c32d8bd11
-    chalk: ^3.0.0
+    "@docusaurus/core": 2.0.0-beta.ff31de0ff
+    "@docusaurus/types": 2.0.0-beta.ff31de0ff
+    "@docusaurus/utils": 2.0.0-beta.ff31de0ff
+    "@docusaurus/utils-validation": 2.0.0-beta.ff31de0ff
+    chalk: ^4.1.1
     eta: ^1.11.0
-    fs-extra: ^9.1.0
+    fs-extra: ^10.0.0
     globby: ^11.0.2
-    joi: ^17.4.0
     lodash: ^4.17.20
+    tslib: ^2.2.0
   peerDependencies:
     react: ^16.8.4 || ^17.0.0
     react-dom: ^16.8.4 || ^17.0.0
-  checksum: 90f95fe9dfa98068d3bd2afdecacc8c772780fc6d0461d05edb2cefd68e5ae8a454872fdcbe868bb96a6054854c171806f3ef284b1bea9be45fe6ebb9f073983
+  checksum: e28da3ea1e078f881cbbbf5d41b7af82f6dd1122d8be5577c888f06ed8ce14b9ddb7407217b71ced581a4865c7264104627aec4e1ce9ebc096597c474265e0be
   languageName: node
   linkType: hard
 
-"@docusaurus/plugin-content-blog@npm:2.0.0-alpha.c32d8bd11":
-  version: 2.0.0-alpha.c32d8bd11
-  resolution: "@docusaurus/plugin-content-blog@npm:2.0.0-alpha.c32d8bd11"
+"@docusaurus/plugin-content-blog@npm:2.0.0-beta.ff31de0ff":
+  version: 2.0.0-beta.ff31de0ff
+  resolution: "@docusaurus/plugin-content-blog@npm:2.0.0-beta.ff31de0ff"
   dependencies:
-    "@docusaurus/core": 2.0.0-alpha.c32d8bd11
-    "@docusaurus/mdx-loader": 2.0.0-alpha.c32d8bd11
-    "@docusaurus/types": 2.0.0-alpha.c32d8bd11
-    "@docusaurus/utils": 2.0.0-alpha.c32d8bd11
-    "@docusaurus/utils-validation": 2.0.0-alpha.c32d8bd11
-    chalk: ^4.1.0
+    "@docusaurus/core": 2.0.0-beta.ff31de0ff
+    "@docusaurus/mdx-loader": 2.0.0-beta.ff31de0ff
+    "@docusaurus/types": 2.0.0-beta.ff31de0ff
+    "@docusaurus/utils": 2.0.0-beta.ff31de0ff
+    "@docusaurus/utils-validation": 2.0.0-beta.ff31de0ff
+    chalk: ^4.1.1
     feed: ^4.2.2
-    fs-extra: ^9.1.0
+    fs-extra: ^10.0.0
```

Comment:
fs-extra v10 does not support node v10 like this repo does. not sure how much of a problem that is?

Reply:
root/node_modules/fs-extra remains in v9 in practice